packages/design-system/src/stories/messaging/Popover.stories.tsx

```diff
@@ -55,7 +54,6 @@ export const FormDisclosureStory = () => (
 	<div style={{ padding: '0.75rem' }}>
 		<Popover
 			aria-label="Custom popover"
-			focusOnDisclosure
```

Comment:
Is this removal on purpose ?

Reply:
Props doesn't exist anymore in Popover component